Paltineanu Rares-Mihai/3.7 - curs5/main.c

```diff
@@ -0,0 +1,61 @@
+/**Given three positive integers, representing a year, a month and a day-of-the-month respectively,
+write a function to compute the number of the day in the year, and the number of days to the end of that year
+*/
+#include <stdio.h>
+#include <stdlib.h>
+int months[] = {0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
+
+int nrOfDays(int y, int m, int d) {
+
+    int nr = 0;
+    for(int i = 1; i < m; i++) {
+        nr += months[i];
+    }
+    nr += d;
+    return nr;
+}
+
+void printfdate(int m, int d) {
+
+    switch(d)
+    {
+        case 1: printf("The number of the 1st of "); break;
+        case 2: printf("The number of the 2nd of"); break;
+        case 3: printf("The number of the 3rd of"); break;
+        default: printf("The number of the %dth of", d);
```

Comment:
I might be nitpicky, but you are missing `21st, 22nd, 23rd and 31st`  😄

Reply:
I forgot about these cases :)))